src/deserialize/mod.rs

```diff
@@ -289,4 +289,20 @@ mod test {
             )]))
         )
     }
+
+    #[test]
+    fn namespaced_maps() {
+        let edn = ":abc{ 0 :val 1 :value}";
```

Comment:
Have you tested with a space between `c` and `{`?

Reply:
I cannot have the space, checked

Reply:
But what does the spec say? I know we don't verify and return errors, but we should accept every valid EDN.

Reply:
I checked the spec

Reply:
What could happen is this `#myapp/Person {:first "Fred" :last "Mertz"}`. It is a different case, tagged map, and it is not really a valid edn, is similar to `#inst` and `#uuid` but needs a reader implemented in the client

Reply:
oh shoot. Okay then

yeah, it is way more featured than JSON.